python/example-seller.py

```diff
@@ -0,0 +1,236 @@
+
+  cancel_all(toCancel)
+  
+  if not toKeep:
+    place_order(pair, newPrice, newOrderVolume, volumeCurrency, otherCurrency)
+
+
+def calculate_price(forexRate, myOrders, allOrders, delta, options):
+  counterOffersIgnorableVolume = Decimal(options['counterOffersIgnorableVolume'])
+  counterOffersIgnorableTotalVolume = Decimal(options['counterOffersIgnorableTotalVolume'])
+  pricePips = Decimal(options['pricePips']) / 10000
+  priceLimitPips = Decimal(options['priceLimitPips']) / 10000
+  othersOrders = [subtract_my_orders_at_price(myOrders, price) for price in allOrders]
+  
+  newPrice = None
+  
+  sumVolume = Decimal('0')
+  for order in othersOrders:
+    if Decimal(order['baseVolume']) <= counterOffersIgnorableVolume:
+      sumVolume = sumVolume + Decimal(order['baseVolume'])
+
+      if sumVolume > counterOffersIgnorableTotalVolume:
+        newPrice = Decimal(order['price']) + delta * pricePips
+        break
+    else:
+      newPrice = Decimal(order['price']) + delta * pricePips
+      break
+
+  if newPrice == None:
+    print('All current offers are either smaller than {0} or their sum is less than {0}'
+      .format(counterOffersIgnorableVolume, counterOffersIgnorableTotalVolume))
+    return None
+
+  distToForex = delta * (forexRate - newPrice)
+  if distToForex < priceLimitPips:
+    newPrice = forexRate - delta * priceLimitPips
+  return newPrice.quantize(FOURPLACES)
+
+
+def calculate_volume(forexRate, newPrice, availBalance, delta, options):
+  currencyVolume = Decimal(options['volumePLN']) / forexRate
+  distToForex = delta * (forexRate - newPrice)
+  if distToForex >= Decimal(options['doubleVolumeLimitPips']) / 10000:
+    currencyVolume = currencyVolume * 2
+
+  return min([currencyVolume, Decimal(availBalance['balanceAll'])]).quantize(TWOPLACES)
+
+
+def difference(list1, list2):
+  return [e for e in list1 if e not in list2]
+
+
+def orders_not_at_price(orders, price): 
+  return filter(lambda order: Decimal(order['price']) != price, orders)
+
+
+def orders_at_price(orders, price): 
+  return filter(lambda order: Decimal(order['price']) == price, orders)
+  totalSoldAtPrice = map(lambda order: Decimal(order['soldAmount'].split(' ')), atPrice)
+  sumAtPrice = sum(totalSoldAtPrice)
+  return sumAtPrice
+
+
+def subtract_my_orders_at_price(myOrders, price):
+  atPrice =  orders_at_price(myOrders, Decimal(price['price']))
+  sellingAtPrice = map(lambda order: Decimal(order['volume']) - Decimal(order['soldAmount'].split(' ')[0]), atPrice)
+  totalMineAtPrice = sum(sellingAtPrice)
+  return {
+    'price': price['price'],
+    'baseVolume': (Decimal(price['baseVolume']) - totalMineAtPrice).quantize(TWOPLACES)
+  }
+
+
+def orders_not_at_volume(orders, volume):
+  return filter(lambda order: Decimal(order['volume']) != volume or order['completion'] > 0, orders)
+
+
+def input_default(prompt, default):
+  return raw_input(prompt.format(default)) or default
```

Comment:
In python3 `raw_input` was renamed to `input`, perhaps you could change shebang to explicit python2? Or `input` + python3